tests/flow/test_effect_schema_desync.py

```
@@ -0,0 +1,186 @@
+        g.query("CREATE (), ()")
+
+        # craft a GRAPH.EFFECT that creates an edge with relation_id = 100
+        # the graph has zero edge schemas - this is the customer's crash
+        payload = _build_effects_payload([
+            ('create_edge', 100, 0, 1),
+        ])
+
+        try:
+            self.conn.execute_command('GRAPH.EFFECT', graph_id, payload)
+        except Exception:
+            pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)